README.md

```diff
@@ -1,70 +1,141 @@
-# Getting Started with Create React App
-# Hello World
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<a name="readme-top"></a>
+![](https://img.shields.io/badge/CRYPTOCURRENCIESAPP-violetpink)
 
-## Available Scripts
+# 📗 Table of Contents
 
-In the project directory, you can run:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-### `npm start`
+# 📖Cryptocurrency Data Explorer
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+<a name="about-project"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+# Cryptocurrency Data Explorer
 
-### `npm test`
+The crypto currency data explorer app is a powerful tool that allows users to delve deep into the world of digital currencies. With real-time updates and advanced analytical tools, this app provides a comprehensive view of the cryptocurrency market, allowing users to make informed decisions about their investments.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+When the homepage is loaded, the user can view a list of various currencies that they can filter based on the currency name. Upon selecting a particular currency, the user is redirected to a details page that retrieves comprehensive information from the API associated with the selected coin.
 
-### `npm run build`
+The application is constructed using React, Redux, and CSS, leveraging the Redux Thunk middleware to manage asynchronous actions and the Fetch API to access data from the NAPI. Moreover, the application is entirely responsive, adjusting to different screen sizes and devices.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+![screenshot](./src/Assets/mobile.homepage.PNG?raw=true "Crypto Mobile App")
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+[PROJECT DOCUMENTATION](https://www.loom.com/share/2d11d6428d4746378f20174565d15e09)
 
```

Comment:
- [ ] Good Job @TaiwoEnoch :+1: Please from the project requirement, in your presentation video, you should highlight a piece of the code that you find interesting or you are proud of making. :pray: